README.md

```diff
@@ -55,6 +55,7 @@ Or, if you're using Bundler, just add the following to your Gemfile:
 * [ePay](http://www.epay.dk/epay-payment-solutions/)
 * [First Data](https://firstdata.zendesk.com/entries/407522-first-data-global-gateway-e4sm-payment-pages-integration-manual)
 * [HiTRUST](http://www.hitrust.com.hk/)
+* [MOLPay](http://www.molpay.com/v2/)
```

Comment:
Other entries here have their countries - for someone reading this README, is it helpful to know what countries a gateway is for? If it's not, don't bother.